.github/actions/deploy-release-v1/action.yml

```diff
@@ -0,0 +1,50 @@
+name: deploy-release-v1
+description: Label all related issues with the package version and create a PR to sync branches
+
+inputs:
+  token:
+    description: 'A GitHub token with the required permissions'
+    required: true
+  project-number:
+    description: 'A project number of the project board'
+    required: false
+    default: '66'
+  head:
+    description: 'A head branch to sync from'
+    required: false
+    default: main
+  base:
+    description: 'A target branch for the created pull request'
+    required: false
+    default: develop
+  pr-team-reviewers:
+    description: 'Reviewers to tag on the created pull request'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
```

Comment:
We should add a note here to note remove this when https://github.com/dequelabs/axe-api-team-public/issues/146 gets picked up as this is should be affected